app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarActivity.java

```diff
@@ -367,6 +403,7 @@ private void deleteCalendarFromGoogle() {
             calendar.set(Calendar.DAY_OF_MONTH, curDay);
             List<CalendarItem> calendarItems = calendarController.getFromDbForDate(new Date(calendar.getTimeInMillis()));
             for (CalendarItem calendarItem : calendarItems) {
+                if (!showCanceledEvents && calendarItem.getStatus().equals("CANCEL")) continue;
```

Comment:
There is no need to use `continue` here - just do the normal `if` with brackets and it should be all good.

Reply:
With kotlin, this would just be `.filter().map()` :see_no_evil:

Reply:
@kordianbruck agree, changed